### PR TITLE
Bookstore Milestone: Add reducers and actions

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Used linters
✅ Set up a React App
✅ Used React components
✅ Used React props
✅ Used React state
✅ Handled events in a React app
✅ Used React Router
✅ Understood the concept of state management
✅ Used store, actions, and reducers in React
✅ Setup a Redux store
✅ Created a slice to save state and reducers

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      state.books = [action.payload, ...state.books];
+    },
+    removeBook: (state, action) => {
+      const indexOfId = state.books.indexOf(action.payload);
+      state.books.splice(indexOfId, 1);
+    },
+  },
+});
+
+export default booksSlice;
```
Comment (Ibizugbe) [inline src/redux/books/booksSlice.js:19]: I suggest you Kindly export `addBook` and `removBook` as actions👍 as these actions will become handy when updating the book state😉.
something like this
```suggestion
export default booksSlice;
export const { addBook, removeBook } = booksSlice.actions
```
``

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @CarlosIgreda 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] Gitflow have been used👍.
- [x] The file structure follows the duck pattern👍.
- [x] The store uses configureStore to create a combined reducer👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (CarlosIgreda) [reply src/redux/books/booksSlice.js:19]: @Ibizugbe, the required changes are done.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @CarlosIgreda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
@NahnahAJ, thanks for your feedback! 🚀

**Pull request merged**